frontend/src/components/common/Calendar/WeeklyEventItems.tsx

```diff
@@ -46,8 +50,8 @@ const DefaultWeeklyEventItem = ({
             borderColor="#D8DDE0"
             alignItems="center"
             centerContent
-            py="1.5rem"
-            px="2rem"
+            py={isMobile ? "1.25rem" : "1.5rem"}

```

Comment:
since you're not doing any conditional rendering, I think all of the `isMobile` can just be 

`py={{base: "1.25rem", lg: "1.5rem"}}` but this is fine!